### ISSUE TITLE
UX suggestion: Large toot window

### ISSUE THREAD

**Issue by lirontocker** [mastodon/mastodon#20799]
### Pitch

Hi there, 

I'd like to suggest a larger toot window for consideration. I believe there are two reasons to consider this: 

1. The toot field is small, making it more challenging to input longer posts
2. Having the toot field alongside the content produces a toot display duplication which is not a bug, but strange UX nonetheless.
3. Moving the toot field into its own window will free up the sidebar for other content, like hashtags.

I've attached the explanations of the issue and wireframes for a proposed solution.

Thanks!
![issue_1](https://user-images.githubusercontent.com/7912787/201984195-0cbec82b-fabf-4384-bbeb-a21ed5c59639.png)
![issue_2](https://user-images.githubusercontent.com/7912787/201984211-a13557ef-8088-4eb1-badc-fffa690434b3.png)
![solution_1](https://user-images.githubusercontent.com/7912787/201984280-ef0d2c66-ed2f-4e61-b757-df0f7beb9eaf.png)
![solution_2](https://user-images.githubusercontent.com/7912787/201984297-615e5da2-fc18-4594-8bdb-b2aab4d0fe1a.png)
![reply_solution_3](https://user-images.githubusercontent.com/7912787/201984320-90c7f72f-8cab-4325-b192-74cebb39d9c8.png)

### Motivation

This improvement could benefit all users of the web interface.

**Comment by smatthie** [comment]
Yes, please. The window is (on a smaller screen like a laptop) not even one single line when there is also a reply and content warning. 
See also #18597.
